packages/web-app-files/src/services/folder/loaderSpace.ts

```diff
@@ -32,15 +32,8 @@ export class FolderLoaderSpace implements FolderLoader {
   }
 
   public getTask(context: TaskContext): FolderLoaderTask {
-    const {
-      store,
-      userStore,
-      spacesStore,
-      router,
-      clientService,
-      configurationManager,
-      capabilityStore
-    } = context
+    const { store, userStore, spacesStore, router, clientService, configStore, capabilityStore } =
+      context
```

Comment:
The TaskContext looks really awkward by now - we should revisit it at some point